mobile/.bazelrc

```diff
@@ -77,6 +77,12 @@ build:mobile-dbg-ios --config=ios
 # https://github.com/envoyproxy/envoy/issues/24977
 build:test-android --define=static_extension_registration=enabled
 
+# Default flags for iOS tests.
+build:test-ios --config=ios
```

Comment:
can we prefix these flags with `mobile-` please

in this case its unlikely to matter, but previously it took a while to unpick ~clashing flags with the base envoy bazelrc - which is why i added the prefix to all in this file - so they cant clash

Reply:
(just noticing that ios doesnt currently have the prefix, but i think it should)

Reply:
done